Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -170,26 +163,30 @@ def StateActivity(self, state, mouse1simulated, mouse2simulated):
             else:
                 self.reward_manager.deliver_reward('dd', 2, self.punishment_time)
             """
+
         elif state == States.WaitForReturn:
             Play(Sounds.Abort)
 
-
         elif state == States.TrialCompleted:
             # Increment the trial number counter
             self.numcompletedtrial += 1
             print("Trial Completed. Number of completed trials: ", self.numcompletedtrial)
             self.trial_logger.log_trial_data(self.numcompletedtrial, "Completed Trial", self.opponent_choice, self.mouse_choice, self.mouse_reward, self.opponent_reward)
 
         elif state == States.TrialAbort:
+            # Increment the trial number counter
+            #self.numcompletedtrial += 1
             # Log that the trial has been aborted
-            print("Trial has been aborted.")  # Or use a logging mechanism if available
+            print("Trial has been aborted.")
             self.opponent_choice = "N/A",
             self.mouse_choice = "N/A",
             self.mouse_reward = "-",
             self.opponent_reward = "-"
             self.trial_logger.log_trial_data(self.numcompletedtrial, "Return Abort", self.opponent_choice, self.mouse_choice,self.mouse_reward, self.opponent_reward)
 
         elif state == States.DecisionAbort:
+            # Increment the trial number counter
+            #self.numcompletedtrial += 1
```

Comment:
i dont think trial should be aborted here

Reply:
I know : )  We had a discussion about it.
That's why I commented it out. We are still not counting the aborted trials. It is there just in case, going forward, we will change our mind. 
For the records,  I still disagree :-). I think an experiment should be defined in terms of total number of trials not number of successful trials. The percentage of successful trials in an experiment, is one of the indications of learnings.

Reply:
ok sure